### PR TITLE
Mobify Analytics

### LINKED ISSUE #9209: Intent to Implement Mobify Analytics as vendor to amp-analytics

### PR DESCRIPTION
**Pull request by kschmidtdev**
- Adds support for Mobify Analytics via tracking pixel

Closes #9209

### CODE REVIEW

**Comment by lannka** [comment]
@kschmidtdev can you add a test in `test/vendor-requests.json`

**Comment by kschmidtdev** [comment]
@lannka Can you include a link to the file you are referring to? There are changes to `extensions/amp-analytics/0.1/test/vendor-requests.json` but I can't find `test/vendor-requests.json`

File: extensions/amp-analytics/0.1/vendors.js [hunk 1]
```diff
@@ -849,6 +849,53 @@ export const ANALYTICS_CONFIG = /** @type {!JSONType} */ ({
     },
   },
 
+  'mobify': {
+    'vars': {
+      'projectSlug': '',
```
Comment (lannka) [inline extensions/amp-analytics/0.1/vendors.js]: empty var can be omitted

**Comment by lannka** [comment]
@kschmidtdev a kind ping. are you still on this?

**Comment by kschmidtdev** [comment]
@lannka yes thank you for the reminder! Hopefully back on this tomorrow 🤞

**Comment by lannka** [comment]
@kschmidtdev a kind ping. are you still on this?

**Comment by kschmidtdev** [comment]
@lannka yes I still plan to resolve this 🙈

**Comment by lannka** [comment]
Closing now. Feel free to reopen it when you are on it again.

**Pull request closed without merging**